mysql-replication/README.md

```diff
@@ -23,12 +23,12 @@ You can deploy the template with Azure Portal, or PowerShell, or Azure cross pla
 * Open Azure Powershell console, and log in by running Add-AzureAccount command.
```

Comment:
This still say to use Add-AzureAccount - maybe just remove it since the command doesn't need to be part of the bullet